packages/drivers/local-driver/src/localDocumentServiceFactory.ts

```diff
@@ -88,8 +87,8 @@ export class LocalDocumentServiceFactory implements IDocumentServiceFactory {
     ): Promise<IDocumentService> {
         ensureFluidResolvedUrl(resolvedUrl);
 
-        const parsedUrl = parse(resolvedUrl.url);
-        const [, tenantId, documentId] = parsedUrl.path ? parsedUrl.path.split("/") : [];
+        const parsedUrl = new URL(resolvedUrl.url);
+        const [, tenantId, documentId] = parsedUrl.pathname ? parsedUrl.pathname.split("/") : [];
```

Comment:
`url.parse().path` is _not_ the same as `new URL().pathname`. The former includes the query string, while the latter does not. This code change is therefor a behavior change, but I don't think it matters because the query isn't parsed. Moreover, similar functions exist elsewhere in our codebase, and in those cases, the existing code already used `.pathname`.